lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -161,15 +153,14 @@ def transform_xy(x, y):
             labels = self.grid_info["lon_labels"]
             labels = [l for l, m in zip(labels, mask) if m]
 
-
         def f1():
-            dd = np.arctan2(yy1b-yy1a, xx1b-xx1a) # angle normal
-            dd2 = np.arctan2(yy2b-yy2a, xx2b-xx2a) # angle tangent
-            mm = ((yy1b-yy1a)==0.) & ((xx1b-xx1a)==0.) # mask where dd1 is not defined
+            dd = np.arctan2(yy1b-yy1a, xx1b-xx1a)  # angle normal
+            dd2 = np.arctan2(yy2b-yy2a, xx2b-xx2a)  # angle tangent
+            mm = ((yy1b-yy1a)==0.) & ((xx1b-xx1a)==0.)  # mask where dd1 is not defined
             dd[mm] = dd2[mm] + np.pi / 2
 
-            #dd += np.pi
-            #dd = np.arctan2(xx2-xx1, angle_tangent-yy1)
+            # dd += np.pi
+            # dd = np.arctan2(xx2-xx1, angle_tangent-yy1)
```

Comment:
should these just go away?

Reply:
The whole block came in commented out with the surrounding code in 4fa9271 so yes.